src/styles/colors.scss

```diff
@@ -25,8 +25,4 @@ $linkforeground: $very-dark-blue;
 $shadow-for-gradient-area: $almost-black;
 $shadow-for-white-column: $very-dark-blue;
 
-// sass-lint:disable no-color-literals
-$blue-gray-gradient: linear-gradient(180deg, #09719b 0%, #09719b 2.36%, #424a52 100%);
-// sass-lint:enable no-color-literals
-
-// TODO: decode how do we import variables by scss import or css-modules composes
+$blue-gray-gradient: linear-gradient(180deg, $dark-blue 0%, $dark-blue 2.36%, $very-dark-gray-blue 100%);
```

Comment:
I replaced colour literals with sass variables using colours that were already defined in this file. So there were no more lint errors. Unless we want to use that colour literals in this line but I don't think it makes sense.